app/src/main/java/com/besome/sketch/ProjectsFragment.java

```diff
@@ -213,19 +230,14 @@ private void toExportProjectActivity(int position) {
         startActivity(intent);
     }
 
+    private void showHideSearchBox(boolean hide) {
+       // projectsSearchView.setVisibility(hide ? View.GONE : View.VISIBLE);
```

Comment:
what?

Reply:
my bad, forgot I was looking at a single commit's changes

Reply:
yea I forgor to commit before adding new stuffs too.
It somehow got left out when deleting